bors.toml

```diff
@@ -1,9 +1,52 @@
+# See https://forum.bors.tech/t/bug-wildcard-status-ignores-1-match/438 
+# for why we need to explicitly list all statuses
+
 status = [
   "Lint",
   "Unit Tests (1.16)",
   "Unit Tests (1.17)",
-  "Integration Tests (1.16)",
-  "Integration Tests (1.17)",
+  "Integration Tests (1.16, integration-mvp)",
+  "Integration Tests (1.16, integration-ghost)",
+  "Integration Tests (1.16, integration-network)",
+  "Integration Tests (1.16, integration-epochs)",
+  "Integration Tests (1.16, integration-access)",
+  "Integration Tests (1.16, integration-collection)",
+  "Integration Tests (1.16, integration-consensus)",
+  "Integration Tests (1.16, integration-execution)",
+  "Integration Tests (1.16, integration-verification)",
+  "Integration Tests (1.17, integration-mvp)",
+  "Integration Tests (1.17, integration-ghost)",
+  "Integration Tests (1.17, integration-network)",
+  "Integration Tests (1.17, integration-epochs)",
+  "Integration Tests (1.17, integration-access)",
+  "Integration Tests (1.17, integration-collection)",
+  "Integration Tests (1.17, integration-consensus)",
+  "Integration Tests (1.17, integration-execution)",
+  "Integration Tests (1.17, integration-verification)",
```

Comment:
this is pretty noisy, it makes me want to move to just a single go version. Thoughts?